website/pages/en/publishing/publishing-a-subgraph.mdx

```diff
@@ -28,6 +28,16 @@ Developers can add GRT signal to their subgraphs. If a subgraph is eligible for
 
 > If your subgraph is eligible for rewards, is recommended that you curate your own subgraph with at least 3,000 GRT (as of April 11th, 2024) in order to attract additional indexers to index your subgraph
 
+The [Sunrise Upgrade Indexer](/sunrise/#what-is-the-upgrade-indexer) ensures the indexing of all subgraphs, signaling GRT on a particular subgraph will draw more indexers to it. This incentivization of additional Indexers through curation aims to enhance the quality of service for queries by reducing latency and enhancing network availability.
```

Comment:
```suggestion
While the [Sunrise Upgrade Indexer](/sunrise/#what-is-the-upgrade-indexer) ensures the indexing of all subgraphs, signaling GRT on a particular subgraph will draw more indexers to it. This incentivization of additional Indexers through curation aims to enhance the quality of service for queries by reducing latency and enhancing network availability.
```
Or something else but basically I feel like there's a link missing between the two parts of this sentence. Or maybe they should just be two sentences?